gnovm/stdlibs/std/native.gno

```diff
@@ -1,5 +1,9 @@
 package std
 
+const (
+	ZeroAddress Address = "g100000000000000000000000000000000dnmcnx"
```

Comment:
```suggestion
	ZeroAddress Address = "g100000000000000000000000000000000dnmcnx" // specific to gno.land
```

Adding a comment to revisit and improve the multichain support of the GNO standard library.

Reply:
https://github.com/gnolang/gno/pull/2401/commits/57617d88095ae940e9fc8356762e15b2136322ea